templates/bitcoin-sample.conf

```diff
@@ -29,9 +14,6 @@ rpcauth=<rpcauth>
 
 # Optimizations
 dbcache=<size>
-maxconnections=40
 maxmempool=300
-maxorphantx=10
+maxconnections=40
 maxuploadtarget=1000
-minrelaytxfee=0.00000001
```

Comment:
Ah, this is actually required for https://github.com/getumbrel/umbrel/issues/96

Reply:
According to the docs:

```
  -minrelaytxfee=<amt>
       Fees (in BTC/kB) smaller than this are considered zero fee for relaying,
       mining and transaction creation (default: 0.00001)
```

So by setting minrelaytxfee=0.00000001 we're saying we relay 1 sat/kb not 1 sat/b.

The default value of 0.00001, is 0.00001 BTC/kb, which is 1000 sat/kb, which is 1 sat / byte.

I don't think we should relay <1 sat/b transactions since that could lead to users broadcasting transactions that never confirm.

As discussed on Keybase, I can remove that change for now until we have time to investigate the fee issue further.